src/main/java/com/crowdin/cli/commands/actions/ListProjectAction.java

```diff
@@ -17,20 +17,26 @@
     private boolean noProgress;
     private String branchName;
     private boolean treeView;
+    private boolean plainView;
 
-    public ListProjectAction(boolean noProgress, String branchName, boolean treeView) {
-        this.noProgress = noProgress;
+    public ListProjectAction(boolean noProgress, String branchName, boolean treeView, boolean plainView) {
+        this.noProgress = noProgress || plainView;
         this.branchName = branchName;
         this.treeView = treeView;
+        this.plainView = plainView;
     }
 
     @Override
     public void act(PropertiesBean pb, Client client) {
         Project project;
         try {
-            ConsoleSpinner.start(RESOURCE_BUNDLE.getString("message.spinner.fetching_project_info"), this.noProgress);
+            if (!plainView) {
```

Comment:
There is no need to use if-statement. `plainView` is already in `noProgress` variable

Reply:
--no-progress is only hiding the actual "progress bar".
the final output would still include "✔️  Fetching project info"
The plainView condition removes that.

See https://paste.myself5.de/axikiluqon.pl for what I mean.